angular-workspace/projects/ni/nimble-angular/rich-text-mention/users/tests/nimble-rich-text-mention-users.directive.spec.ts

```diff
@@ -94,14 +104,15 @@ describe('NimbleRichTextMentionUsers', () => {
         @Component({
             template: `
                 <nimble-rich-text-editor>
-                    <nimble-rich-text-mention-users [pattern]="pattern" #users></nimble-rich-text-mention-users>
+                    <nimble-rich-text-mention-users [pattern]="pattern" [button-label]="buttonLabel" #users></nimble-rich-text-mention-users>
```

Comment:
nit: Apparently it works either way, but it seems like for the property binding tests, we should be using the property name rather than the attribute name.
```suggestion
                    <nimble-rich-text-mention-users [pattern]="pattern" [buttonLabel]="buttonLabel" #users></nimble-rich-text-mention-users>
```